api/v1beta1/rabbitmqcluster_types.go

```diff
@@ -243,6 +243,13 @@ type StatefulSetSpec struct {
 	// Template.
 	// +optional
 	UpdateStrategy *appsv1.StatefulSetUpdateStrategy `json:"updateStrategy,omitempty" protobuf:"bytes,7,opt,name=updateStrategy"`
+
+	// The minimum number of seconds for which a newly created DaemonSet pod should
```

Comment:
Should this be DaemonSet? I think it's StatefulSet in our case.

Reply:
Yes strange, I copied the comment from the Kubernetes code, now I see they removed completely the word! Fixed!